jaegertracing/jaegertracing_test.go

```diff
@@ -18,6 +18,7 @@ import (
 type mockSpan struct {
 	tracer   opentracing.Tracer
 	tags     map[string]interface{}
+	logs     map[string]interface{}
```

Comment:
What's the reason to separate tags and logs?
The name `logs` seems a little misleading here. Seems like only body and error message are put there currently.